### PR TITLE
Check to see if config file exists to stop error log

### PR DESCRIPTION
**Pull request by MatthewSandfordImprobable**
Inside IsManualWorkerConnectionSet we check for the existence of the LaunchConfigPath file and error if it does not exist.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp [hunk 1]
```diff
@@ -464,8 +464,11 @@ bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 		}
 	}
 
+	FString LaunchConfigPath = GetPrimaryLaunchConfigPath();
+	TUniquePtr<FArchive> ConfigFile(IFileManager::Get().CreateFileReader(*LaunchConfigPath));
+
 	TArray<FString> WorkersManuallyLaunched;
-	if (IsManualWorkerConnectionSet(GetPrimaryLaunchConfigPath(), WorkersManuallyLaunched))
+	if (ConfigFile != nullptr && IsManualWorkerConnectionSet(LaunchConfigPath, WorkersManuallyLaunched))
```
Comment (improbable-valentyn) [inline SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp:471]: Could we use `IFileManager::Get().FileExists` instead?

File: SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp [hunk 2]
```diff
@@ -464,8 +464,11 @@ bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 		}
 	}
 
+	FString LaunchConfigPath = GetPrimaryLaunchConfigPath();
+	TUniquePtr<FArchive> ConfigFile(IFileManager::Get().CreateFileReader(*LaunchConfigPath));
```
Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp:468]:
```suggestion
	TUniquePtr<FArchive> ConfigFile(IFileManager::Get().FileExists(*LaunchConfigPath));
```

File: SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp (hunk 1 above)
Comment (improbable-valentyn) [reply SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp:471]: Also, it seems like we're doing the same here inside `IsManualWorkerConnectionSet`. Perhaps we should just not error if the file isn't there.

**Review by improbable-valentyn — CHANGES_REQUESTED** [review]
I feel like this would be better accomplished by adding an early out in `IsManualWorkerConnectionSet` that doesn't print an error if the file doesn't exist.

**Comment by MatthewSandfordImprobable** [comment]
> This would be better accomplished by adding an early out in `IsManualWorkerConnectionSet` that doesn't print an error if the file doesn't exist.

So just remove the error log.

**Review by improbable-valentyn — APPROVED** [review]

**Review by improbable-danny — APPROVED** [review]

**Pull request merged**